modules/openapi-generator/src/main/resources/Java/libraries/retrofit/ApiClient.mustache

```diff
@@ -57,9 +58,11 @@ public class ApiClient {
             {{#hasAuthMethods}}
             Interceptor auth;
             {{#authMethods}}if ("{{name}}".equals(authName)) {
-            {{#isBasic}}
-                auth = new HttpBasicAuth();
-            {{/isBasic}}
+            {{#isBasic}}{{#isBasicBasic}}
+              auth = new HttpBasicAuth();
+            {{/isBasicBasic}}{{^isBasicBasic}}
+              auth = new HttpBearerAuth("{{scheme}}");
```

Comment:
@davidwcarlson I'll file another PR to clean up the indentation using either 2-space or 4-space to make it consistent with the rest of the files.

Reply:
UPDATE: https://github.com/OpenAPITools/openapi-generator/pull/2521 merged into master to improve the code indention